frameworks/hdfs/tests/auth.py

```diff
@@ -52,7 +52,7 @@ def get_principal_to_user_mapping() -> str:
     we need to create an appropriate mapping to test authorization functionality.
     :return: A base64-encoded string of principal->user mappings
     """
-    rules = ["RULE:[2:$1@$0](^hdfs@.*$)s/.*/hdfs/", "RULE:[1:$1@$0](^nobody@.*$)s/.*/nobody/"]
+    rules = ["RULE:[2:$1@$0](^hdfs@.*$)s/.*/hdfs/"]
```

Comment:
Note that this rule effectively overrides the rules that we define since *ALL* our principals match this pattern.

Reply:
When looking at the actual `core-site.xml` the added rules look like this `hdfs/journal-0-node.testintegrationhdfs.autoip.dcos.thisdcos.directory@LOCAL)s/.*/nobody/`. I already tried removing the other rule as well but then the tests start failing. So I think this rule is an addition.

Reply:
If the rules we add with `nobody` don't work, then we need to ensure that our rules look like:
```
RULE:[2:$1/$2@$0](hdfs/journal-0-node.testintegrationhdfs.autoip.dcos.thisdcos.directory@LOCAL)s/.*/hdfs/
```
Remember that this has to map to the HDFS super user which is `hdfs`...

Reply:
Is there anything I have to change? Since the integration test passes without the need for the superuser rule so it is not required for the initial setup at least.

Reply:
It may be good to check that this passes on a strict cluster. I'll start a build tomorrow.